src/AdaptiveCardContainer.tsx

```diff
@@ -173,8 +179,25 @@ class AdaptiveCardContainer extends React.Component<Props, State> {
             }
 
             if (renderedCard) {
+                if (!this.props.interactive) {
+                    const hyperlinks = renderedCard.querySelectorAll('a');
+                    const inputs = renderedCard.querySelectorAll('button, input, select, textarea');
+
+                    [].forEach.call(inputs, (input: HTMLInputElement) => {
```

Comment:
For consistency's sake:

on line 203 you use the convention `Array.prototype.forEach.call()`, but here you use `[].forEach.call()`

It would be nice to stick to one convention / pattern